src/style.css

```diff
@@ -2,2 +1,90 @@
+@font-face {
+  font-family: Roboto;
+  src: url('/fonts/Roboto-Medium.ttf') format('truetype');
+  font-weight: 500;
+}
+
+* {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: Roboto, sans-serif;
+}
+
+header {
+  display: flex;
+  align-items: center;
+  width: 100%;
+  background-color: white;
+  padding: 0 16px;
+  position: relative;
+  height: 60px;
+}
+
+.logo {
+  margin-left: 34px;
+  margin-top: 5px;
+}
+
+nav {
+  margin-left: auto;
+  margin-right: 34px;
+}
+
+nav ul {
+  display: flex;
+  list-style: none;
+  height: 100%;
+}
+
+nav ul li {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100%;
+}
+
+nav ul li:not(:first-child) {
+  margin-left: 20px;
+}
+
+.nav-link {
+  text-decoration: none;
+  color: #000;
+  font-size: 12px;
+  line-height: 60px;
+  text-align: center;
+  position: relative;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100%;
+  font-weight: 500;
+  text-transform: uppercase;
+}
+
+.nav-link:hover {
+  color: #00acdc;
+}
+
+.nav-link.is-active {
+  color: #00acdc;
+}
+
+.nav-link.is-active::after {
+  content: '';
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  width: 100%;
+  height: 4px;
+  background-color: #00acdc;
+  border-radius: 2px;
+}
+
+[data-qa='hover'] {
+  color: black;
```

Comment:
The `data-qa='hover'` selector is used to apply a specific style. Ensure that this attribute is used consistently in the HTML to avoid unexpected styling issues.